runelite-client/src/main/java/net/runelite/client/plugins/osrswikia/OSRSWikiaPlugin.java

```diff
@@ -0,0 +1,43 @@
+package net.runelite.client.plugins.osrswikia;
+
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Provides;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.api.events.CommandExecuted;
+import net.runelite.client.util.LinkBrowser;
+
+import javax.inject.Inject;
+
+@PluginDescriptor(
+        name = "OSRS Wikia",
+        enabledByDefault = false
+)
+public class OSRSWikiaPlugin extends Plugin {
+    private final String BASEURI = "http://oldschoolrunescape.wikia.com/wiki";
+    
+    @Inject
+    private OSRSWikiaConfig config;
+    
+    @Provides
+    OSRSWikiaConfig getConfig(ConfigManager configManager) {
+        return configManager.getConfig(OSRSWikiaConfig.class);
+    }
+    
+    @Subscribe
+    public void onCommand(CommandExecuted commandExecuted) {
+        if(!commandExecuted.getCommand().equals("wiki") || !config.allowWikiCommand())
```

Comment:
Check if wiki command is allowed first, and make command equals incasesensitive.